client/src/components/Search/Search.jsx

```diff
@@ -32,7 +40,63 @@ export default function Search() {
   };
 
   //handle local routing
-  function routerHandler() {
+  // async function routerHandler() {
+  //   await getSelectedData();
+  //   return history.push("/list");
+  // }
+
+  //test----- fetching data from database 
+  // async function fetchAllData(){
+  //   console.log("I'm in fetchAllData!"); 
+  // const req = axios.get("http://localhost:4000/api/availability/data");
+  // const res = await req; 
+  // console.log(res); 
+  // const data = res.data; 
+  // console.log(data);
+  // }
+  // fetchAllData();
+
+  //test-----fetching selected data from database 
+  async function fetchSelectedData(){
+  const req = axios.get("http://localhost:4000/api/availability?day=Monday&address_city=Roppongi&start_hour=10&end_hour=11");
+  const res = await req; 
+  const data = res.data; 
+  }
+  fetchSelectedData();
+
+  //handle selected data 
+  //get entered data from imput 
+  async function getSelectedData(){
+    // parse the location
+    const selectedLocation = location.split(",")[0]
+
+    const date = new Date(selectedDate)
+    
+    // parse day from selected date
+    const week = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+    const selectedDay = week[date.getDay()];
+    console.log(selectedDay)
```

Comment:
all console logs should be removed before pushing to master

Reply:
Thx! I forget to remove them. Sorry for that.